resources/js/web3_init.js

```diff
@@ -53,18 +53,23 @@ StatusHttpProvider.prototype.prepareRequest = function () {
 };
 
 function sendAsync(payload, callback) {
-    var messageId = callbackId++;
-    callbacks[messageId] = callback;
-    if (typeof StatusBridge == "undefined") {
-        var data = {
-            payload: JSON.stringify(payload),
-            callbackId: JSON.stringify(messageId),
-            host: this.host
-        };
+    if (payload.method == "eth_coinbase"){
+        callback(null, web3Response(payload, currentAccountAddress));
```

Comment:
added callback for async "eth_coinbase"

Reply:
can we do the same for `eth_accounts` and `net_version` to be consistent with sync methods?

Reply:
sounds reasonable @mandrigin